packages/frontapp/src/app/components/CategoriesList/index.tsx

```diff
@@ -63,7 +64,7 @@ const CategoriesList: FC<ICategoriesListProps> = ({ allCategories }) => {
   return (
     <CategoriesWrapper>
       <Categories flex="2">
-        <Header>Books</Header>
+        <Header>{t('Books')}</Header>
```

Comment:
This translation won't work when you switch to Russian (check how we use translation in other components)